txpipe/calibrate.py

```diff
@@ -53,10 +54,21 @@ def run(self):
         extra_cols = [c for c in self.config["extra_cols"] if c]
         subtract_mean_shear = self.config["subtract_mean_shear"]
 
+        with self.open_input("shear_catalog", wrapper=True) as f:
+            bands = f.get_bands()
+
+        shear_prefix = self.config["shear_prefix"]
+        # this is the names of the columns in the input catalog
+        mag_cols_out = [f"mag_{b}" for b in bands] + [f"mag_err_{b}" for b in bands]
+        mag_cols_in = [f"{shear_prefix}{c}" for c in mag_cols_out]
```

Comment:
It looks like in the example the shear catalog input only contains 3 bands (the bands used in shear measurement maybe?) but for RAIL we will probably want all bands? 
Maybe either we should make sure the shear_catalog input has all available bands, or make the photometry_catalog magnitudes an input to this stage?

Reply:
It's still a bit up in the air whether there will be a full set of bands available in the shear catalog. For metadetect the mags have to be measured as part of the calibration process so that we can find the impact on the shear selection bias of cutting on them for tomography, so it's unfortunately not as simple as taking them from the photo catalog. So we might have to do shear tomography and n(z) estimation with a restricted set of bands.

Reply:
oh ok. I hadn't realized that. Then the current setup is good